relay-server/src/actors/store.rs

```diff
@@ -325,20 +465,24 @@ struct UserReportKafkaMessage {
     event_id: EventId,
 }
 
-#[derive(Debug, Serialize)]
+#[derive(Clone, Debug, Serialize)]
 struct SessionKafkaMessage {
     org_id: u64,
     project_id: ProjectId,
     session_id: Uuid,
     distinct_id: Uuid,
+    quantity: u32,
     seq: u64,
     received: f64,
     started: f64,
     duration: Option<f64>,
-    status: SessionStatus,
+    status: &'static str,
```

Comment:
Please leave this `SessionStatus`.

Reply:
The problem here is that it will allow `errored` to be set for individual session updates. I mean sure, we can do that, maybe warn and switch that to be `exited` in that case.